README.md

```diff
@@ -29,3 +29,6 @@
   * could still use some love, but isn't so bad anymore
 
 
+### Troubleshooting:
+
+* On OSX El Captain, gem installation could fail with a permission error. See: http://stackoverflow.com/questions/32659348/operation-not-permitted-when-on-root-el-capitan-rootless-disabled
```

Comment:
Wow, that's like going nuclear ... Not sure I want that to be the recommended solution as it creates a security risk.
What's needed is [rvm](https://rvm.io) and making sure you have a ruby version + a gemset for the website generation.